.github/workflows/test_code.yml

```diff
@@ -1,44 +1,36 @@
 name: Code Tests
 
 on:
-  push:
-    branches:
-    - master
   pull_request:
-    branches:
-    - master
+  push:
+    branches: [master]
 
 jobs:
   run:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
-        python-version: [3.6, 3.9]
```

Comment:
we should still support version 3.6, so I'd like to still test for it.

Reply:
I'm just testing to see which ones fail the selenium tests. We will need to skip those tests on those py versions.